gpustack/assets/model-catalog.yaml

```diff
@@ -511,6 +511,48 @@
       huggingface_repo_id: unsloth/gemma-2-{size}b-it
       replicas: 1
       backend: vllm
+- name: Phi4 mini
+  description: Phi-4-mini-instruct is a lightweight open model built upon synthetic data and filtered publicly available websites - with a focus on high-quality, reasoning dense data. The model belongs to the Phi-4 model family and supports 128K token context length. The model underwent an enhancement process, incorporating both supervised fine-tuning and direct preference optimization to support precise instruction adherence and robust safety measures.
+  home: https://azure.microsoft.com/en-us/products/phi
+  icon: /static/catalog_icons/microsoft.png
+  capabilities:
+    - 128k context
+  categories:
+    - llm
+  sizes:
+    - 3.8
+  licenses: 
+    - mit
+  release_date: "2025-02-18"
+  templates: 
+    - quantizations:
+        - Q2_K_L
+        - Q3_K_M
+        - Q4_K_M
+        - Q5_K_M
+        - Q6_K
+      source: huggingface
+      huggingface_repo_id: unsloth/microsoft_Phi-4-mini-instruct-GGUF
+      huggingface_filename: "*-{quantization}*.gguf"
+      replicas: 1
+      backend: llama-box
+      cpu_offloading: true
+      distributed_inference_across_workers: true
+    - quantizations:
+        - Q8_0
+        - BF16
+      source: huggingface
+      huggingface_repo_id: unsloth/Phi-4-mini-instruct-GGUF
+      huggingface_filename: "*.{quantization}*.gguf"
```

Comment:
Combine different quantizations of the model to one item. We can set `"*{quantization}*.gguf"` to make them match.